docs/how-to/add-mesh-support-to-your-charm.md

```diff
@@ -58,17 +71,11 @@ class MyCharm(CharmBase):
     self._mesh = ServiceMeshConsumer(
         self,
         policies=[
-            Policy(
-                relation="metrics-endpoint",  # On the metrics-endpoint relation
-                endpoints=[                   # allow a related application to access...
-                    Endpoint(
-                        paths=["/metrics"],        # these specific paths
-                        ports=[HTTP_LISTEN_PORT],  # on these specific ports
-                        methods=[Method.get],      # using only these methods
-                    ),
-                ],
+            UnitPolicy(
+                relation="metrics-endpoint",  # On the metrics-endpoint relations
+                ports=[HTTP_PORT],  # allow a related application to access this charm's individual units on these specific ports
```

Comment:
tiny nit - most other things in our docs cite AppPolicy first before UnitPolicy.  Can we move this example down below app?